pkg/ui/src/views/shared/components/sortedtable/index.tsx

```diff
@@ -120,16 +141,71 @@ export class SortedTable<T> extends React.Component<SortedTableProps<T>, {}> {
     },
   );
 
+  constructor(props: SortedTableProps<T>) {
+    super(props);
+    // TODO(vilterp): use a LocalSetting instead so the expansion state
+    // will persist if the user navigates to a different page and back.
+    this.state = {
```

Comment:
We can get rid of the constructor override by writing the init this way: https://github.com/cockroachdb/cockroach/blob/304150d8010b05cd5bd106e6e6b63a89c2835403/pkg/ui/src/views/shared/components/expandableString/index.tsx#L26